src/Domain/CoordinateMaps/EquiangularMap.cpp

```diff
@@ -0,0 +1,150 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/EquiangularMap.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+
+namespace CoordinateMaps {
+
+EquiangularMap::EquiangularMap(const double A, const double B, const double a,
+                               const double b) noexcept
+    : A_(A),
+      B_(B),
+      a_(a),
+      b_(b),
+      length_of_domain_(B - A),
+      length_of_range_(b - a),
+      linear_jacobian_(length_of_range_ / length_of_domain_),
+      linear_inverse_jacobian_(length_of_domain_ / length_of_range_) {}
+
+template <typename T>
+std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, 1> EquiangularMap::
+operator()(const std::array<T, 1>& xi) const noexcept {
+  return {{0.5 * (a_ + b_ +
+                  length_of_range_ * tan(M_PI_4 * (-B_ - A_ + 2.0 * xi[0]) /
+                                         length_of_domain_))}};
+}
+
+template <typename T>
+std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, 1>
+EquiangularMap::inverse(const std::array<T, 1>& x) const noexcept {
+  return {{0.5 *
+           (A_ + B_ +
+            length_of_domain_ *
+                atan((-a_ - b_ + 2.0 * x[0]) / (length_of_range_)) / M_PI_4)}};
+}
+
+template <typename T>
+Tensor<std::decay_t<tt::remove_reference_wrapper_t<T>>,
+       tmpl::integral_list<std::int32_t, 2, 1>,
+       index_list<SpatialIndex<1, UpLo::Up, Frame::NoFrame>,
+                  SpatialIndex<1, UpLo::Lo, Frame::NoFrame>>>
+EquiangularMap::jacobian(const std::array<T, 1>& xi) const noexcept {
+  return Tensor<std::decay_t<tt::remove_reference_wrapper_t<T>>,
+                tmpl::integral_list<std::int32_t, 2, 1>,
+                index_list<SpatialIndex<1, UpLo::Up, Frame::NoFrame>,
+                           SpatialIndex<1, UpLo::Lo, Frame::NoFrame>>>{
+      linear_jacobian_ * M_PI_4 *
```

Comment:
you might as well store `linear_jacobian_ * M_PI_4` instead of `linear_jacobian_`